src/Features/Core/Portable/CodeFixesAndRefactorings/AbstractFixAllGetFixesService.cs

```diff
@@ -0,0 +1,116 @@
+        }
+
+        public async Task<ImmutableArray<CodeActionOperation>> GetFixAllOperationsAsync(
+            IFixAllContext fixAllContext, bool showPreviewChangesDialog)
+        {
+            var codeAction = await GetFixAllCodeActionAsync(fixAllContext).ConfigureAwait(false);
+            if (codeAction == null)
+            {
+                return ImmutableArray<CodeActionOperation>.Empty;
+            }
+
+            return await GetFixAllOperationsAsync(
+                codeAction, showPreviewChangesDialog, fixAllContext.ProgressTracker, fixAllContext.State, fixAllContext.CancellationToken).ConfigureAwait(false);
+        }
+
+        private static async Task<CodeAction> GetFixAllCodeActionAsync(IFixAllContext fixAllContext)
+        {
+            var fixAllKind = fixAllContext.State.FixAllKind;
+            var functionId = fixAllKind switch
+            {
+                FixAllKind.CodeFix => FunctionId.CodeFixes_FixAllOccurrencesComputation,
+                FixAllKind.Refactoring => FunctionId.Refactoring_FixAllOccurrencesComputation,
+                _ => throw ExceptionUtilities.UnexpectedValue(fixAllKind)
+            };
+
+            using (Logger.LogBlock(
+                functionId,
+                KeyValueLogMessage.Create(LogType.UserAction, m =>
+                {
+                    m[FixAllLogger.CorrelationId] = fixAllContext.State.CorrelationId;
+                    m[FixAllLogger.FixAllScope] = fixAllContext.State.Scope.ToString();
+                }),
+                fixAllContext.CancellationToken))
+            {
+                CodeAction action = null;
+                try
+                {
+                    action = await fixAllContext.FixAllProvider.GetFixAsync(fixAllContext).ConfigureAwait(false);
+                }
+                catch (OperationCanceledException)
+                {
+                    FixAllLogger.LogComputationResult(fixAllKind, fixAllContext.State.CorrelationId, completed: false);
+                }
+                finally
+                {
+                    if (action != null)
+                    {
+                        FixAllLogger.LogComputationResult(fixAllKind, fixAllContext.State.CorrelationId, completed: true);
+                    }
+                    else
+                    {
+                        FixAllLogger.LogComputationResult(fixAllKind, fixAllContext.State.CorrelationId, completed: false, timedOut: true);
+                    }
+                }
+
+                return action;
+            }
+        }
+
+        protected static ImmutableArray<CodeActionOperation> GetNewFixAllOperations(ImmutableArray<CodeActionOperation> operations, Solution newSolution, CancellationToken cancellationToken)
+        {
+            var result = ArrayBuilder<CodeActionOperation>.GetInstance();
```

Comment:
```suggestion
            using var _ = ArrayBuilder<CodeActionOperation>.GetInstance(out var result);
```

Reply:
doesn't seem to be valid code? no overloads match

Reply:
i pushed the change to your branch.